src/interface/src/app/plan/area-notes/area-notes.component.html

```diff
@@ -1,4 +1,6 @@
-<div class="notes">
+<div class="notes-header" *appFeatureFlag="'new_planning_area'">Notes</div>
+<!-- old component section-->
```

Comment:
Might be a bit late for this, but wondering if its an easier/cleaner approach to have a separate component for the new version, and feature flag that where used

Reply:
true, let me see what's possible